build.sh

```diff
@@ -5,8 +5,8 @@ set -e
 CWD=`pwd`
 
 build_mac64() {
-    echo "Building mac64 components..."
-    # sh build_sms.sh
+    echo "Building mac64 components..." 
+    sh build_doc.sh M64
```

Comment:
Change adds reference to guid_doc.sh shell script but can't see any such file in the commit logs or this pull request.